File: Server/src/schema/Transportation/baggage.ts
import { Field, InputType, Int, ObjectType } from "type-graphql";
import { BaseEntity, Column, Entity, PrimaryGeneratedColumn } from "typeorm";

@Entity()
@ObjectType()
export default class Baggage extends BaseEntity {
    @Field(() => Int!)
    @PrimaryGeneratedColumn()
    baggageID!: number;

    @Field(() => Int!)
    @Column()
    tripID!: number;

    @Field(() => Int!)
    @Column()
    transportTypeID!: number;

    @Field(() => Int!)
    @Column()
    amount!: number;

    @Field(() => Int!)
    @Column()
    weight!: number;

    constructor() {
        super();

        this.baggageID = 0;
        this.tripID = 0;
        this.transportTypeID = 0;
        this.amount = 1;
        this.weight = 1;
    }
}

File: Assets/GraphQL/Model/Transportation/Baggage.cs
using GraphQL.Helpers;

namespace GraphQL.Model {
	public class Baggage {
		[GqlSelection("baggageID")] public int BaggageID { get; private set; }
		[GqlSelection("tripID")] public int TripID { get; private set; }
		[GqlSelection("transportTypeID")] public int TransportTypeID { get; private set; }
		[GqlSelection("amount")] public int Amount { get; private set; }
		[GqlSelection("weight")] public int Weight { get; private set; }
	}
}